packages/open-truss/src/signals/index.ts

```diff
@@ -28,6 +32,10 @@ export function getSignalsType(
   return getSignalAndValueShape(possibleZodObject)?.signal
 }
 
+export function getSignalsShapeFromSignal(signal: Signal): SignalsZodType {
```

Comment:
How about `signalValueShape`? It's shorter and mentions `valueShape` which I think is a good thing

Reply:
9031dca680437f7f701cebf779faffa2ce82b413